dask_planner/src/sql/parser_utils.rs

```diff
@@ -14,20 +17,22 @@ impl DaskParserUtils {
             } => {
                 let identities: Vec<String> = name.0.iter().map(|f| f.value.clone()).collect();
 
-                assert!(identities.len() <= 2 && !identities.is_empty());
+                assert!(!identities.is_empty());
```

Comment:
This assertion is redundant since the empty case is covered in the `match` below where we return an `Err` instead of causing a panic.